cli/js/io.ts

```diff
@@ -49,13 +49,95 @@ export interface SeekerSync {
   seekSync(offset: number, whence: SeekMode): number;
 }
 
+class RedableStreamIOReader implements Reader {
```

Comment:
Typo, this should probably be `ReadableStreamIOReader`.

Reply:
Indeed, good catch. Thanks!